src/App.jsx

```diff
@@ -32,30 +57,27 @@ class App extends React.Component {
           recipe: data.meals[0],
         });
       });
+    this.getSlider = this.getSlider.bind(this);
```

Comment:
Je comprends pas bien ce que cette ligne fait içi, elle ne devrait pas plutot etre après "this.getRecipe = this.getRecipe.bind(this);" (ligne 24) ?

Reply:
C'est ce que javais fait au debut mais ça ne marchait pas je comprenez pas pourquoi